cirq/google/engine/engine.py

```diff
@@ -65,8 +65,8 @@ def __init__(self, message):
         super().__init__(message)
 
 
-# Quantum programs to run can be specified as circuits or schedules.
-TProgram = Union[circuits.Circuit, schedules.Schedule]
+# Quantum programs to run can be specified as circuits.
+TProgram = 'cirq.Circuit'
```

Comment:
think we could just use cirq.Circuit for these types?

Reply:
Done.